compiler/crates/relay-codegen/src/printer.rs

```diff
@@ -192,6 +192,7 @@ pub struct JSONPrinter<'b> {
     eager_es_modules: bool,
     js_module_format: JsModuleFormat,
     top_level_statements: &'b mut TopLevelStatements,
+    elide: bool,
```

Comment:
nit: maybe a more descriptive name? `should_skip_printing_nulls`?

Reply:
👍 . I might make the feature flag name `skip_printing_nulls` so it's consistent with this too